src/hooks/useFilterCriteria.ts

```diff
@@ -19,6 +18,12 @@ interface useFilterCriteriaReturns {
   loading: boolean;
 }
 
+export const orderByNewestTimeToFinish = (a: any, b: any) =>

```

Comment:
Not sure if the intention is not to use a lot of types. But `(a: ProposalsExtended, b: ProposalsExtended)` sounds appropriate

Reply:
Nice spot, I was just copy pasting. Fixed now